go/vt/sqlparser/sql.y

```diff
@@ -6890,66 +7102,91 @@ non_reserved_keyword:
 | X509
 | YEAR
 
-// Reserved keywords that cause grammar conflicts in some places, but are safe to use as column name / alias identifiers.
-// These keywords should also go in reserved_keyword.
-// TODO: The ones commented out here cause shift/reduce conflicts but need to be column name safe
-column_name_safe_reserved_keyword:
-  ACCOUNT
-| ARRAY
-| ATTRIBUTE
-| AUTO_INCREMENT
-| AVG
-| BIT_AND
-| BIT_OR
-| BIT_XOR
-| CONNECTION
-| COUNT
-| END
-| ERRORS
+// non_reserved_keywords that can't go in non_reserved_keywords for some reason
```

Comment:
Rather get to the bottom on this and have a single list. You'll have to debug the output from goyacc to see where the conflict is coming from